comms/EclMeta.cpp

```diff
@@ -0,0 +1,75 @@
+#include "StdAfx.h"
+
+#include "EclMeta.h"
+
+CEclMeta::CEclMeta()
+{
+}
+
+void CEclMeta::LoadMetaData(IEclCC *eclcc)
```

Comment:
CEclMeta should not "know" about IEclCC.  You could simply pass in an array of the root folders here instead.